content/source/docs/enterprise/vcs/troubleshooting.html.md

```diff
@@ -84,3 +84,101 @@ This often happens when the VCS connection has been somehow broken: it might hav
 The domain name for Terraform Enterprise changed on 02/22 at 9AM from atlas.hashicorp.com to app.terraform.io. If the OAuth client was originally configured on the old domain, using it for a new VCS connection can result in this error.
 
 The fix is to update the OAuth Callback URL in your VCS provider to use app.terraform.io instead of atlas.hashicorp.com.
+
+### Debugging Certificate Errors on Private Terraform Enterprise
+
+When debugging failures of VCS connections due to certifcate errors, it's sometime helpful to run the OpenSSL command within the TFE container to get more information.
+
+Attach a bash session to the PTFE container:
+
+```
+docker exec -it ptfe_atlas sh -c "stty rows 50 && stty cols 150 && bash"
+```
+
+Then run the openssl command, using the certificate from the container, and you'll be able to see the SSL connection, and any errors that come up.
+
+The certificate will be available under `/tmp/cust-ca-certificates.crt` in the container:
+
+```
+$ openssl s_client -showcerts -CAfile /tmp/cust-ca-certificates.crt -connect git-server-hostname:443
```

Comment:
Let's be consistent about whether to put a sigil at the start of the line, if it's a line standing by itself. 

...but I've been doing GUI stuff so long I can't remember what our standard practice is at the moment! Let's go with "not" for now.

Reply:
Done! 👍

Reply:
We're actually not very consistent about this - it would probably be worth going through the docs at some point to clean up :)